PendingReleaseNotes.md

```diff
@@ -41,4 +53,22 @@
   An upgraded operator will create the necessary service account, role, and role bindings to enable this.
 - API
   - The API pod now uses RBAC permissions that are scoped only to the namespace it is running in.
-  An upgraded operator will create the necessary service account, role, and role bindings to enable this.
\ No newline at end of file
+  An upgraded operator will create the necessary service account, role, and role bindings to enable this.
+- Filesystem
+  - The Rook Flexvolume uses the `mds_namespace` option to specify a cephFS. This is only available on Kernel v4.7 or newer. On older kernel, if there are more than one filesystems in the cluster, the mount operation could be inconsistent. See this [doc](/Documentation/k8s-filesystem.md#consume-the-file-system).
```

Comment:
should this link be pointing to the common-problems.md page entry instead?

Reply:
I thought about it. They both have the same information. But since this is a ReleaseNote doc, i thought it would be cleaner to point it to the k8s-filesystem doc.

I see the common-problem doc for users who are running into issues and trying to debug things. Whereas releasenote docs are more official and cleaner.

But i dont have any objections in linking the common-problem docs from the release-notes doc.

Reply:
i'm OK with linking to actual documentation instead of the common problems in the release notes, but the link isn't very direct.  this note is talking about a specific issue and doesn't link directly to information about that issue.  instead, it links to a general section about using filesystems (that happens to have a relevant note at the end of it).

i would prefer a more direct link that goes right to information about this issue.

Reply:
ahh ok.. good point.. I was trying to figure out how to link a `note`. I will convert the `note` to a header.